ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -2098,7 +2098,6 @@ objects:
             --template-file "05_masters.json" \
             --parameters masterIgnition="$MASTER_IGNITION" \
             --parameters sshKeyData="$SSH_PUB_KEY" \
-            --parameters privateDNSZoneName="${CLUSTER_NAME}.${base_domain}" \
```

Comment:
Removing this will cause the job to fail for earlier OpenShift versions that still require the parameter.

Reply:
I removed privateDNSZoneName in 05_masters.json at https://github.com/openshift/installer/pull/5375.

Reply:
That change will only affect jobs running against the master branch of the installer. This template is run against other branches of the installer as well.

Reply:
Agree. Then QE will close this PR, we will only modify upi ARM templates and set a default value there to work with different version of install.